streams/src/main/java/org/apache/kafka/streams/StreamsBuilder.java

```diff
@@ -144,7 +144,7 @@ public synchronized <K, V> KStream<K, V> stream(final String topic,
      * @return a {@link KStream} for the specified topics
      */
     public synchronized <K, V> KStream<K, V> stream(final Collection<String> topics) {
-        return stream(topics, Consumed.with(null, null, null, null));
+        return stream(topics, Consumed.with(null, null));
```

Comment:
With the new overload, compiler could not resolve this any longer -- switching the a different overload to fix it.